ui/src/components/easter/hero/HeroContainer.js

```diff
@@ -1,12 +1,152 @@
-import { Text } from "@chakra-ui/react";
+import { Container, Text, Box, Button, Flex, VStack, Heading, Link, Icon, RiArrowRightLine, RiChat1Line, Faq, Grid, background, LinkOverlay, Image, _hover } from "@chakra-ui/react";
+import { SimpleGrid, LinkBox } from "@chakra-ui/react";
 
+//import easterTheme from "../easterTheme";
 const HeroContainer = () => {
   return (
-    <>
-      <Text textStyle='FogtwoNo5' fontSize={38}>
-        2022
-      </Text>
-    </>
+    <Box>
```

Comment:
![Screenshot 2022-03-15 at 22 33 14](https://user-images.githubusercontent.com/33679483/158405959-1da13675-09d6-4a16-8154-e4b646b82ab8.png)

can we add some vertical padding between the Easter text and the button? refer to the figma mobile design

Reply:
![Screenshot 2022-03-15 at 22 34 44](https://user-images.githubusercontent.com/33679483/158406693-fa7b42d0-db59-41f9-bae7-4f5b5ecab974.png)
also the GIF and the Easter/Button should be all the way to the edge of the container, refer to figma design for desktop

also i think need more paddingY between GIF, EasterText/Button, picture